examples/provider-install-verification/main.tf

```diff
@@ -182,3 +182,111 @@ resource "stytch_rbac_policy" "b2b_rbac_policy" {
     }
   ]
 }
+
+# Invalid resources below. Uncomment to test config validation
```

Comment:
Is this the only way to test config validation? I see there is `b2bsdkconfig_test.go`, could we add unit tests for these cases?

Reply:
I'll look around to see if there's a specific way to test validation of *invalid* configs. The examples from implementing the plugin were all for validating "correct" resources.

Reply:
Found a way to test these! Added them into the acceptance tests.